AppController/djinn.rb

```diff
@@ -1037,15 +1032,17 @@ def set_parameters(djinn_locations, database_credentials, app_names, secret)
       set_log_level(Logger::INFO)
     end
 
-
-
     begin
       @options['zone'] = JSON.load(@options['zone'])
     rescue JSON::ParserError
+      Djinn.log_info("Fail to parse 'zone': ignoring it.")
```

Comment:
@menivaitsi we can put is a warning: right now I see a lot of them since the tools always pass something in (even empty). I think we should have a way to parse it out if it's not a cloud deployment.